packages/config/src/configDefaults.ts

```diff
@@ -35,8 +35,10 @@ export const getInitialConfig = ({
     ethpm: {
```

Comment:
in line with the non-breaking-ness suggestion below, we could add a `version` field here that would control the delegation to the correct v1/v2 package.

Reply:
so then to switch it, users would provide this sort of configuration:

```javascript
module.exports = {
  /* ... rest of truffle-config.js ... */
  ethpm: {
    version: "2",
    /* ... rest of ethpm config ... */
  }
}
```